Vision/README.md

```diff
@@ -128,6 +128,19 @@ We will do this by:
 	including building and maintaining respected relationships 
 	with governments and businesses for providing credible advice.
 
+# Status
+
+Status of This Document
+This document reflects the consensus of the AB at the time of publication. It will continue to evolve and the AB will issue updates as often as needed. The intent is for this document to become a W3C Statement.
```

Comment:
It's not obvious to me that this is the case, in particular because the membership of the AB has changed over the time the document has been developed. (It isn't necessarily not the case - I think it hasn't been tested properly for the entirety of the document so we don't know).